R/utils.R

```diff
@@ -44,3 +44,13 @@ redirect_id <- function(id, url) {
   url <- header$url
   drive_extract_id(url)
 }
+
+get_mime_type <- function(x) {
+  if (!(x %in% c("document", "spreadsheet", "folder", "presentation",
+                 "form", "drawing", "script"))) {
+    message(glue::glue("Input was not a Google Drive type: {x}"))
+    invisible(NULL)
+  } else {
+  paste0("application/vnd.google-apps.", x)
+  }
+}
```

Comment:
I think basically the current `ext` column in `.drive$mime_tbl` should also be valid input. As in `get_mime_type("pdf")` returns `application/pdf`.